susemanager/bin/server-migrator.sh

```diff
@@ -22,18 +22,19 @@ echo
 spacewalk-service stop
 mv /etc/zypp/repos.d /etc/zypp/repos.d.old
```

Comment:
We should remove `/etc/zypp/repos.d.old/` first. People that migrated from 15.1 to 15.2 will already have it.

Reply:
```suggestion
rm -rf /etc/zypp/repos.d.old
mv /etc/zypp/repos.d /etc/zypp/repos.d.old
```